### PR TITLE
Split API Blueprint Source Map from the AST

### PR DESCRIPTION
**Pull request by zdne**
This PR does not change the structure of AST or Source Maps

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,62 @@
...
-
-API Blueprint Source Map is 1-on-1 with API Blueprint AST and represents the source maps for each of the nodes in the AST tree.
+For the definition of API Blueprint AST Source Map see the [API Blueprint AST Source Map definition][Source Map Definition].
 
 Converting API Blueprint to AST and its serialization is the task of API Blueprint Parser – [Snow Crash](https://github.com/apiaryio/snowcrash) or one of its [bindings](https://github.com/apiaryio/snowcrash#bindings). Reverse process from AST (serialization) to API Blueprint is also possible thanks to the [Matter Compiler](https://github.com/apiaryio/matter_compiler).
 
-## What?
-
-**This document is intended for authors of tools using API Blueprint**. If you are looking for a way to describe your Web API proceed directly to [API Blueprint](https://github.com/apiaryio/api-blueprint).
+## Do not like to design APIs in JSON or YAML?
```
Comment (kylef) [inline README.md]: I think it sounds/looks better if you use "don't" instead of "do not".

File: README.md [hunk 2]
```diff
@@ -1,73 +1,62 @@
 ![logo](https://raw.github.com/apiaryio/api-blueprint/master/assets/logo_apiblueprint.png)
 
-# API Blueprint AST & Source Map Serialization Media Types
-### API Blueprint's contract with machines
+# API Blueprint AST
+### JSON & YAML face of the API Blueprint
+API Blueprint AST is JSON or YAML, machine-friendly, face of API Blueprint suitable for use in [tools](http://apiblueprint.org/#tooling) consuming (or producing) API Blueprint.
 
-This document defines serialization formats of [API Blueprint](http://apiblueprint.org) abstract syntax tree, or AST for short and Source Map tree.
+This document defines serialization format for [API Blueprint](http://apiblueprint.org) abstract syntax tree, or AST for short.
```
Comment (kylef) [inline README.md]: I think either one of the following instead:

- This document defines **the** serialization format for...
- This document defines serialization format**s** for...

**Comment by zdne** [comment]
@kylef fixed, thanks!

**Comment by kylef** [comment]
:shipit:

**Pull request merged**